src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from 'react'
+import { createRoot } from 'react-dom/client'
+import App from './App.tsx'
+import './index.css'
```

Comment:
**praise**: Excellent use of React 18's createRoot API for rendering the application.